src/callback.jl

```diff
@@ -8,11 +12,11 @@ macro callback(ex)
 
 	setter = Symbol("Set", name)                       # SetFooCallback
 	libsetter = Expr(:quote, Symbol("glfw", setter))   # glfwSetFooCallback
-	wrapper = gensym(string(name, "Wrapper"))          # FooCallbackWrapper
+	wrapper = Symbol(string('_', name, "Wrapper"))     # FooCallbackWrapper
```

Comment:
How or why does `gensym` prevent static compilation? Is there documentation somewhere that talks about it?

Reply:
That's what this comment was about: 

>removes non predictable name of callback wrapper for snoop compile (since we control the namespace, this should be safe)

If I snoop compile this, it will catch different names from when it runs the static compilation. Snoop compile is needed to actually generate the precompile commands!